includes/definitions/discovery/hpe-mapdu.yaml

```diff
@@ -163,7 +163,7 @@ modules:
                     -
                         oid: 'pdu3OutletName'
                         op: '=='
-                        value: '00 00 00 00 00 00 00 00'
+                        value: '........'
```

Comment:
this is a quite ugly representation of "unprintable chars" tho

Reply:
Indeed, but It does fix polling oids with no mib.